scripts/lib/CIME/XML/pes.py

```diff
@@ -69,18 +69,18 @@ def _find_matches(self, grid_nodes, grid, compset, machine, pesize_opts, overrid
         pesize_choice = None
         max_points = -1
         pes_ntasks, pes_nthrds, pes_rootpe, other_settings = {}, {}, {}, {}
-        pe_select = []
+        pe_select = None
```

Comment:
empty list is not a valid value for this variable. it's used as a root=pe_select later on